lib/fixup.rb

```diff
@@ -0,0 +1,14 @@
+# Add fixups to be run during crew update here.
+
+# remove deprecated directory
+FileUtils.rm_rf "#{HOME}/.cache/crewcache/manifest"
+
+# Fix missing $PATH not added in install.sh
+if !system("grep -q '$PATH' #{CREW_PREFIX}/etc/env.d/path") || system("grep -q '1.36.4' #{CREW_PREFIX}/etc/env.d/path")
```

Comment:
Not sure I follow the logic here.  We haven't merged a commit with `1.36.4` yet?

Reply:
It was only there during testing... it would have fixed your issue...

Reply:
We don't actually have to merge this, do we?

Reply:
I can take it out... but it is nice to have the code in there as an example of fixing up an issue targeted to a version of a file...

Reply:
If we leave it in, I will cringe every time I examine the code here in the future. :woozy_face: